CloneTest/Quicksort.cs

```diff
@@ -0,0 +1,76 @@
+﻿namespace CloneTest
```

Comment:
Polecam skorzystać z konstrukcji `namespace CloneTest;`. Pozwoli ci to usunąć jedno zagnieżdżenie w kodzie